posthog/hogql/constants.py

```diff
@@ -92,6 +92,9 @@ def get_breakdown_limit_for_context(limit_context: LimitContext) -> int:
 class HogQLQuerySettings(BaseModel):
     model_config = ConfigDict(extra="forbid")
     optimize_aggregation_in_order: Optional[bool] = None
+    use_query_cache: Optional[bool] = None
+    query_cache_ttl: Optional[int] = None
+    allow_experimental_analyzer: Optional[bool] = None
```

Comment:
I added this because Clickhouse doesn't support anything but equality clauses in `ON` statements without it. I used it at first for testing, but then my optimization actually gets rid of the need for it. Will remove for now. 

However, I noticed that Clickhouse switches to it as the default in 2024.03 (March 2024 release). We're on the December 2023 release at the moment. It does have some compatibility issues - I noticed that there are some queries we're running that aren't quite proper that the old analyzer allows.